lib/DatePickerItem.tsx

```diff
@@ -49,27 +49,30 @@ const DatePickerItem: FC<DatePickerItemProps> = ({
 
   useEffect(() => {
     currentIndex.current = MIDDLE_INDEX;
+    console.log('set MIDDLE_INDEX');
     setStateTranslateY(MIDDLE_Y);
     setMarginTop(0);
-    setDates(iniDates({ step, type, value }));
+    dates.current = iniDates({ step, type, value });
   }, [step, type, value]);
 
-  const updateDates = (direction: Direction) => {
+  const updateDates = (direction: Direction, additionalStep: number = 0) => {
+    const movingStep = step + additionalStep;
```

Comment:
Click can change date over 2step in an interaction. So I added `additionalStep` and change `updateDates` logic.